mathesar/api/db/viewsets/columns.py

```diff
@@ -182,6 +186,12 @@ def partial_update(self, request, pk=None, table_pk=None):
                     )
                 else:
                     raise base_api_exceptions.MathesarAPIException(e)
+            except InvalidStringTruncation as e:
+                raise database_api_exceptions.InvalidTypeOptionAPIException(
+                    e,
+                    message='Invalid string truncation length.',
```

Comment:
It would be good to improve this message. Am I correct that the underlying error there is an attempt to cast some column of text-like values to a `VARCHAR(n)` or `CHAR(n)` where the length is too short? If so, that would be a more helpful message. (I.e., "the requested string length is too short" or the like)

Reply:
> Am I correct that the underlying error there is an attempt to cast some column of text-like values to a VARCHAR(n) or CHAR(n) where the length is too short?

You are correct, I'll write a better message.